src/rules/labelPositionRule.ts

```diff
@@ -25,15 +25,15 @@ export class Rule extends Lint.Rules.AbstractRule {
 class LabelPosWalker extends Lint.RuleWalker {
     private isValidLabel: boolean;
 
-    public visitLabeledStatement(node: ts.LabeledStatement): void {
-        var statement = node.statement;
+    public visitLabeledStatement(node: ts.LabeledStatement) {
+        const statement = node.statement;
         if (statement.kind !== ts.SyntaxKind.DoStatement &&
             statement.kind !== ts.SyntaxKind.ForStatement &&
             statement.kind !== ts.SyntaxKind.ForInStatement &&
             statement.kind !== ts.SyntaxKind.WhileStatement &&
             statement.kind !== ts.SyntaxKind.SwitchStatement) {
```

Comment:
might as well update the boolean operator alignment here as well (since this commit does it in other places)